_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+
+This will first trigger a mediation process, between you, the landlord, and the Préfecture. The goal will be to settle the dispute so as to avoid going to court over it.
+If you go this way, then you will likely want to be a member of ASLOCA so they can provide legal advice and represent you the day of the mediation. 
+
+⚠️It may happen that the landlord simply doesn’t come to the mediation. In that case, you will be allowed by the préfecture to take the landlord to court. If so, make sure you have a case before proceeding and seek legal advice, as the process will likely last long and be very costly.
+
+In any case, after starting a procedure against your landlord, you are protected from eviction for a period of 3 years[^3ans]. A resiliation of your lease within that period may be considered as retaliation, so it is unlikely that this will happen.
+
+## Challenging the initial rent
+Similarly to a rent increase, if you spot something in the lease agreement that seems illegal, then you can challenge the initial rent within 30 days after signing the lease. The challenge is justified if one of the following conditions is met: 
+The rent is increased by more than 10% compared to the previous tenant[^10%]
+You signed the lease under duress. This does not necessarily mean you need to prove that you were held at gunpoint when signing the contract, it is justified when there are too few available apartments in your area. More specifically, if the 3-year average vacancy rate in your city is less than 1.5% (which is the case almost everywhere), that qualifies as duress. In Lausanne, the average vacancy rate is around 0.5%[^vacancy]
+
+## Lease fees
+When applying for a flat, it is common that you have to fill out and sign a form for the rental agency, which states that you will pay a fee (typically around 200 CHF) in case your application is selected and you choose not to sign the lease. This is a common scare tactic from agencies to try and prevent people applying to many places and only choosing once they get accepted. Under Swiss law, one can retract from negotiations at any point in time until the contract is signed, so you cannot be asked to pay a fee simply from stopping negotiations. If you refuse an apartment you applied to and the agency sends you an invoice for this fee, you should ask them to justify which costs *you* specifically incurred them. If they cannot provide that (and they probably won’t), then they have no legal basis to ask for that fee[^withdrawal1][^withdrawal2].
+
+Similarly, some agencies may ask for an administrative fee to be paid by the tenant for establishing the contract. This should also not be paid by the tenant[^rulv]
```

Comment:
It would be great if you had concrete templates of how to refuse or challenge such fees.

Reply:
True ! You can also just ignore that invoice, and they should eventually drop it. This is what happened in our case when moving out of Lausanne.